src/algorithms/nncluster.jl

```diff
@@ -0,0 +1,30 @@
+"""
+    NearestNeighborCluster(p, n)
+
+Create a random cluster nearest-neighbour neutral landscape model with 
+values ranging 0-1. `p` sets the density of original clusters, and `n`
+sets the neighborhood for clustering (see `?label` for neighborhood options)
+"""
+struct NearestNeighborCluster <: NeutralLandscapeMaker
+    p::Float64
+    n::Symbol
+    function NearestNeighborCluster(p::Float64 = 0.5, n::Symbol = :rook) 
+        @assert p > 0
+        @assert n ∈ (:rook, :queen, :diagonal)
+        new(p,n)
+    end
+end
+
+function _landscape!(mat, alg::NearestNeighborCluster)
+    _landscape!(mat, NoGradient())
```

Comment:
It would be more consistent with the python code to use `rand!` here, i.e. allow 0-1 rescaling first. Honestly not really sure what is preferable but my impulse was this.